src/current/cockroachcloud/export-metrics.md

```diff
@@ -214,11 +216,118 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
 
 </section>
 
+<section class="filter-content" markdown="1" data-scope="prometheus-metrics-export">
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID:
+
+	1. Visit the CockroachDB {{ site.data.products.cloud }} console [cluster page](https://cockroachlabs.cloud/clusters).
+	1. Click on the name of your cluster.
+	1. Find your cluster ID in the URL of the single cluster overview page: `https://cockroachlabs.cloud/cluster/{your_cluster_id}/overview`. It should resemble `f78b7feb-b6cf-4396-9d7f-494982d7d81e`.
```

Comment:
Suggest changing slightly to be more explicit:

"The ID should resemble ..."

Reply:
Updated